core/templates/components/state-editor/state-editor-properties-services/state-name.service.ts

```diff
@@ -42,14 +42,14 @@ export class StateNameService {
   /**
    * @param {string} value - Memento of the state name to be set to.
    */
-  setStateNameSavedMemento(stateName: string): void {
+  setStateNameSavedMemento(stateName: string | null): void {
```

Comment:
Do we need the `null` here? Are there some places where we call `setStateNameSavedMemento` with `null`?

Reply:
yes, `setStateNamedSavedMemento` is being called with `null` here in `state-name.service.ts`
```typescript
init(): void {
    this.setStateNameSavedMemento(null);
    this.setStateNameEditorVisibility(false);
}
```